pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -686,16 +688,22 @@ private boolean checkCanRemovePendingAcksAndHandle(PositionImpl position, Messag
         return false;
     }
 
+    private boolean checkCanRemovePendingAcksAndHandleWithOwnedConsumer(PositionImpl position,
```

Comment:
```suggestion
    private boolean canRemovePendingAcksAndHandleWithOwnedConsumer(PositionImpl position,
```

I don't know who named `checkCan`, but it seems `canXxx` is more direct than `checkCan`.